source/assets/javascripts/all.js

```diff
@@ -3,9 +3,15 @@
 //= require 'lib/_masonry.pkgd'
 //= require 'lib/_handlebars-v4.0.5'
 //= require 'lib/_ramda'
+//= require 'lib/_raphael-min.js'
+//= require 'lib/_jquery.mapael.js'
+//= require 'lib/_world_countries.js'
+//= require 'lib/_jquery.mousewheel.min.js'
```

Comment:
I considered it, and thought we didn't want to. It used to be like that earlier, but not nice and in layout. I'll do this.